templates/hash.html

```diff
@@ -5,7 +5,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8">
-    <meta http-equiv="Referrer-Policy" content="no-referrer, strict-origin-when-cross-origin">
+    <meta name="referrer" content="no-referrer, strict-origin-when-cross-origin">
```

Comment:
@mozfreddyb: for example this was an error; there's no `http-equiv="Referrer-Policy"`